src/locales/es/battler-tags.json

```
@@ -69,5 +69,6 @@
   "cursedLapse": "¡{{pokemonNameWithAffix}} es víctima de una maldición!",
   "stockpilingOnAdd": "¡{{pokemonNameWithAffix}} ha reservado energía por {{stockpiledCount}}ª vez!",
   "disabledOnAdd": "¡Se ha anulado el movimiento {{moveName}}\nde {{pokemonNameWithAffix}}!",
-  "disabledLapse": "¡El movimiento {{moveName}} de {{pokemonNameWithAffix}} \n ya no está anulado!"
+  "disabledLapse": "¡El movimiento {{moveName}}\nde {{pokemonNameWithAffix}} ya no está anulado!",
```

```suggestion
  "disabledLapse": "¡El movimiento {{moveName}} de {{pokemonNameWithAffix}} ya no está anulado!",
```

I don't want to risk overflowing to 3rd line here